### DISCUSSION TITLE
Best tool for copy&pasting rectangular areas form PDF files?

### DISCUSSION THREAD

**Discussion by aSemy** [red6/pdfcompare#110] [Q&A] [answered]
Firstly, thanks for this tool! 

My problem: I need to process hundreds of PDF files - a daunting task. 

Is there a tool where I can open a PDF, drag a rectangle, press ctrl+c (which copies `x1,y1,x2,y2` to the clipboard)? Then I can just paste the coords into the exclusions file.

Open source tools are preferred.

(bonus points if it can do image files too - I also have to process image files)

**Comment by finsterwalder** [comment]
When you start the UI by starting class de.redsix.pdfcompare.Main you can mark ignore areas and write that to a file.

**Comment by aSemy** [comment, marked as answer]
Thanks @finsterwalder! I figured out how to open the GUI and it's exactly what I wanted. I now realise what you mean, and what the README explains about running the 'full' jar

For anyone else who was confused about how to open up the GUI, here are the steps:

1. Go to https://search.maven.org/artifact/de.redsix/pdfcompare
2.  Select the newest/highest version, currently that's [1.1.60](https://search.maven.org/artifact/de.redsix/pdfcompare/1.1.60/jar)
3. In the top left, click 'Downloads', then 'full.jar' 
![image](https://user-images.githubusercontent.com/897017/130835829-1d7ef2a3-92d5-4626-b702-12bcde4bb3ba.png)
4. This will download `pdfcompare-1.1.60-full.jar`. 
5. Now you can run `java -jar pdfcompare-1.1.60-full.jar` (no arguments) to open the GUI
